docs/specs/restore.yml

```diff
@@ -3,9 +3,25 @@
 inputs:
   docfx.yml: |
     dependencies:
-      docfx-test-dependencies: https://github.com/docascode/docfx-test-dependencies
+      docfx-test-dependencies1: https://github.com/docascode/docfx-test-dependencies
+      docfx-test-dependencies2: https://github.com/docascode/docfx-test-dependencies
 restores:
   git/github.com/docascode/docfx-test-dependencies/9ab39126f6d6642131084e741ca6206989db505a/dep.md: DEP
   git/github.com/docascode/docfx-test-dependencies/5736374c0a142252063bbedafe5d98db495e263d/dep.md: child-DEP
+  restore_lock/01. Restore dependencies with their children-*-lock.json: |
```

Comment:
>lock [](start = 62, length = 4)

I suppose this is a single file containing all transitive dependencies, like package-lock.json ?

Reply:
currently no requirement to put all transitive dependencies in one docset lock file, so let's just do that in this way and change it later once we got a requirement for that.

---
In reply to: [201566709](https://github.com/dotnet/docfx/pull/3042#discussion_r201566709) [](ancestors = 201566709)